spec/triggers/aws-cloudwatch.md

```diff
@@ -0,0 +1,129 @@
+# AWS Cloudwatch Trigger
+
+This specification describes the `aws-cloudwatch` trigger for Azure Storage Queue.
+
+```yaml
+triggers:
+  - type: aws-cloudwatch
+    metadata:
+      # Required: namespace
+      namespace: AWS/SQS
+      # Required: Dimension Name
+      dimensionName: QueueName
+      dimensionValue: keda
+      metricName: ApproximateNumberOfMessagesVisible
+      targetMetricValue: "2"
+      minMetricValue: "0"
+      # Required: region
+      awsRegion: "eu-west-1"    
+```
+
+The `connection` value is the name of the environment variable your deployment uses to get the connection string. This is usually resolved from a `Secret V1` or a `ConfigMap V1` collections. `env` and `envFrom` are both supported.
```

Comment:
You are referring to a `connection` but this is not in the do above, can you please update this?

Reply:
It should describe what the parameters are for folks to configure in the trigger auth in order to use the scaler as well